### PR TITLE
Bump cyclic dependencies.

### PR DESCRIPTION
**Pull request by iclanton**

### CODE REVIEW

File: apps/api-extractor/package.json [hunk 1]
```diff
@@ -52,8 +52,8 @@
   },
```
Comment (octogonz) [inline apps/api-extractor/package.json:52]: The `cyclicDependencyProjects` setting was renamed to `decoupledLocalDependencies`.

- Let's update our `rush.json` settings
- Let's rename the PR title to "Bump decoupled dependencies" -- going forward, we'll use that naming

**Review by octogonz — APPROVED** [review]

File: libraries/heft-config-file/src/ConfigurationFile.ts [hunk 2]
```diff
@@ -767,20 +767,20 @@ export class ConfigurationFile<TConfigurationFile> {
...
             ) {
               throw new Error(
-                `Issue in processing configuration file property "${propertyName}". ` +
+                `Issue in processing configuration file property "${String(propertyName)}". ` +
                   `Primitive types cannot be provided when the inheritance type is set as "${InheritanceType.merge}"`
               );
             } else if (Array.isArray(propertyValue) || Array.isArray(parentPropertyValue)) {
               throw new Error(
-                `Issue in processing configuration file property "${propertyName}". ` +
+                `Issue in processing configuration file property "${String(propertyName)}". ` +
```
Comment (octogonz) [inline libraries/heft-config-file/src/ConfigurationFile.ts:783]: What is `"${String(propertyName)}"`?

File: apps/api-extractor/package.json (hunk 1 above)
Comment (iclanton) [reply apps/api-extractor/package.json:52]: That change in Rush hasn't been published yet.

File: libraries/heft-config-file/src/ConfigurationFile.ts (hunk 2 above)
Comment (iclanton) [reply libraries/heft-config-file/src/ConfigurationFile.ts:783]: TS gives a warning that `propertyName` might be a `symbol`, and using that in a template string throws. Changed this to `.toString()`

**Pull request merged**